src/containers/magasin/panel/page/index.js

```diff
@@ -514,7 +526,13 @@ class StorageUnitContainer extends Component {
                             onClickCancel={() => hashHistory.goBack()}
                           />
                         </Row>
-                      </Grid>
+                        <Row>
+                          <Col sm={8} />
+                          <Col sm={4}>
+                            {this.renderLastChangeData('updatedBy', 'updatedDate', this.props.unit)}
```

Comment:
Remove arguments 'updatedBy', 'updatedDate'. There is no point in sending in keys for updatedBy and updateDate when those keys can be defined/used in the method.